mailing_list/models.py

```diff
@@ -179,11 +190,12 @@ class MailingList(models.Model):
     ACCESS_LEVEL_STAFF = 'staff'
 
     canvas_course_id = models.IntegerField()
+    course_settings = models.ForeignKey(CourseSettings, null=True)
     section_id = models.IntegerField(null=True)
     access_level = models.CharField(max_length=32, default='members')
     created_by = models.CharField(max_length=32)
     modified_by = models.CharField(max_length=32)
-    date_created = models.DateTimeField(blank=True, default=timezone.now)
+    date_created = models.DateTimeField(auto_now_add=True)
```

Comment:
(we will also have to remove the modified date update line in `api.set_access_level()`)